templates/welcome_page.html

```diff
@@ -1,18 +1,82 @@
 <!doctype html>
+<html lang="en">
 <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width", initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie-edge">
     <title>
         autocomplete
     </title>
-<!--    <link rel="stylesheet" type="text/css" href="style.css">-->
+    <!-- <link rel="stylesheet" type="text/css" href="./public/css/index.css"> -->
+    <!--Bootstrap CSS -->>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
 </head>
 
 <body>
 
-<h1>    Welcome to use the auto-complete system</h1>
+</html>
```

Comment:
I think this end tag should be put in the end?

Reply:
done